test/libsolidity/semanticTests/revertStrings/unknown_sig_no_fallback.sol

```diff
@@ -3,7 +3,7 @@ contract A {
 }
 // ====
 // EVMVersion: >=byzantium
-// revertStrings: debug
 // compileViaYul: also
+// revertStrings: debug
 // ----
-// (): hex"00" -> FAILURE, hex"08c379a0", 0x20, 41, "Unknown signature and no fallbac", "k defined"
+// (): hex"00" -> FAILURE, 3963877391197344453575983046348115674221700746820753546331534351508065746944, 862718293348820473429344482784628181556388621521298319395315527974912, 0x29556e6b6e6f776e207369676e617475726520616e64206e6f2066616c, 49023610809053150916515914125811666384002407880190069586813210542196895252480, 26436266249043104754341746750903573776499820331372108504574805934080
```

Comment:
This one is weird why it translated like this, when it nicely handles other error string cases.

Reply:
Maybe it only handles strings with <=32 characters correctly for error strings?

Reply:
failure cases should handle the first 4 bytes differently and then take a look at `output[4:]` split by 32 byte chunks.

Reply:
This one is pretty bad actually (and all the similar ones in this PR too, there are like ~15 of them). It produces non-deterministic results - the output is different every time I run isoltest. I can't even just let isoltest run to completion because  when it gets to one of these, it starts spinning in an infinite loop of updates.

Looking closer at the output, it's one slot longer than the original expectation and this last slot contains garbage. It looks like a bug in isoltest to me.

Reply:
I spent an afternoon trying to fix string handling in isoltest and managed to track it down and fix some things, but turned out it can't really be improved without a major redesign. Too bad I didn't document all my findings :(

Because of this I think we just have to close this PR.